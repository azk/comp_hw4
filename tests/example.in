NIS x =4 NIS;
AGORA y = 100 AGORA;
if(x>y)
{
	print "ok";
}


