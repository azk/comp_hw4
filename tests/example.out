-- end scope --
-- end scope --
var:x:0:NIS
var:y:1:AGORA
